Components/ListLogos/_style.scss

```diff
@@ -0,0 +1,100 @@
+[is='flynt-list-logos'] {
+  padding: $component-spacing-mobile 0;
+
+  @media (min-width: $breakpoint-desktop) {
+    padding: $component-spacing-desktop 0;
```

Comment:
there is now an additional variable `$component-spacing-tablet` that you can add here